docs/concepts/04_credentials/02_ctypes.md

```diff
@@ -2,6 +2,11 @@
 id: ctypes
 title: CTypes
 ---
+import CodeBlock from '@theme/CodeBlock';
+
+<!-- Taken from https://github.com/webpack-contrib/raw-loader/issues/91#issuecomment-648830498 -->
+import ctypeSchema from '@site/scripts/out/ctype-schema.json.raw!=!raw-loader!@site/scripts/out/ctype-schema.json';
```

Comment:
what does the `from` do here? Would it not have been enough to just `import ctypeSchema from '!raw-loader...'`

Reply:
Apparently there is a bug with loading raw JSON files, as mentioned in the link I added in the comment. It does not work if the file is loaded "regularly".